cargo/jam/internal/image.go

```diff
@@ -56,3 +57,59 @@ func FindLatestImage(uri string) (Image, error) {
 		Version: versions[len(versions)-1].String(),
 	}, nil
 }
+
+func FindLatestBuildImage(runURI, buildURI string) (Image, error) {
```

Comment:
Making sure I understand this - this function is to get the build image that shares the same tag as the run image?

Reply:
Yes!